paddle/fluid/operators/detail/grpc_client.cc

```diff
@@ -247,6 +247,8 @@ void GRPCClient::Proceed() {
     } else {
       LOG(ERROR) << "var: " << c->var_h_.String()
                  << " grpc error:" << c->status_.error_message();
+      PADDLE_ENFORCE(false, "Get var from pserver error!context:%s",
```

Comment:
Maybe you can just change the previous ```LOG(ERROR) to LOG(FATAL)```?

Reply:
Thanks.